src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1430,8 +1430,8 @@ def add_hostname(cmd, resource_group_name, webapp_name, hostname, slot=None):
     if slot is None:
```

Comment:
while we are on this code - may be we should update this block to use generic operation instead of the if slot NONE: code block.

Reply:
`generic_site_operation` supports only one extra parameter. In our case we should send 2 as `hostname` and `binding`. Check [here](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/appservice/_appservice_utils.py#L12) for reference.